uwblueprint/planet-read
# Set up dummy test grading page

## Notion ticket link



[Ticket](https://www.notion.so/uwblueprintexecs/Set-up-dummy-Test-Grading-platform-4235faf216e044199d6cd828f110bdf8)



## Implementation description

- copied `ReviewPage.tsx` and modified/removed parts that are not relevant to story test grading



## Steps to test

1. Go to any review page and change the url to `grade/x/y`
2. Verify that the page looks okay

![test grading](https://user-images.githubusercontent.com/45080644/145755598-c7549d2e-700b-4886-a199-38cdd8276f02.PNG)




## What should reviewers focus on?

- is the styling okay?

## Checklist
